core/src/main/groovy/noe/rhel/server/workspace/WorkspaceHttpdTomcatsBaseOS.groovy

```diff
@@ -46,22 +47,42 @@ class WorkspaceHttpdTomcatsBaseOS extends WorkspaceMultipleTomcats{
     httpd.updateConfSetBindAddress(hostIpAddress)
 
     copyModulesIfMissing(httpd)
+    copyConfIfMissing(httpd)
     workspaceTomcat.prepare(true) //always true due to running with JWS 5 `ewsVersion >= new Version('3.1.0-DR0')`
 
     createAdditionalTomcatsWithRegistrations(numberOfAdditionalTomcats)
     originalTomcatHosts.putAll(originalTomcatHostsIpAddresses())
   }
 
   private void copyModulesIfMissing(Httpd httpd) {
-    File modulePath = new File("${httpd.getServerRoot()}/modules/mod_jk.so")
-    File sclModulePath = new File("${DefaultProperties.HTTPD_SCL_ROOT}/usr/lib64/httpd/modules/mod_jk.so")
-    if (!modulePath.exists() && sclModulePath.exists()) {
-      log.info("Copiing file ${sclModulePath} to modules folder of httpd ${modulePath}.")
-      modJkModules.add(modulePath)
-      JBFile.copyFile(sclModulePath, modulePath)
+    List<String> moduleNames = ["mod_jk", "mod_advertise", "mod_cluster_slotmem", "mod_manager", "mod_proxy_cluster"]
+    moduleNames.each() { String moduleName ->
+      File modulePath = new File("${httpd.getServerRoot()}/modules/${moduleName}.so")
+      File sclModulePath = new File("${DefaultProperties.HTTPD_SCL_ROOT}/usr/lib64/httpd/modules/${moduleName}.so")
+      if (!modulePath.exists() && sclModulePath.exists()) {
+        log.info("Copying file ${sclModulePath} to modules folder of httpd ${modulePath}.")
+        moduleFiles.add(modulePath)
+        JBFile.copyFile(sclModulePath, modulePath)
+      }
+      if (!modulePath.exists()) {
+        throw new FileNotFoundException("Module file is missing. ${modulePath}")
+      }
     }
-    if (!modulePath.exists()) {
-      throw new FileNotFoundException("Mod_jk module file is missing. ${modulePath}")
+  }
+
+  private void copyConfIfMissing(Httpd httpd) {
```

Comment:
Comment this method would be beneficial, goal of it seems non standard.

Reply:
Commented, present in next commit

Reply:
Wouldn't be useful to link JIRA? IIRC there is some jira, here this is reported as unexpected structure

Reply:
Added links:
   * @link https://issues.jboss.org/browse/JBCS-730
   * @link https://issues.jboss.org/browse/JBCS-734